engine/common/requester/engine.go

```diff
@@ -404,6 +410,11 @@ func (e *Engine) process(originID flow.Identifier, message interface{}) error {
 }
 
 func (e *Engine) onEntityResponse(originID flow.Identifier, res *messages.EntityResponse) error {
```

Comment:
There is no efficient way to check for unrequested entities, we are skipping them anyway [here](https://github.com/onflow/flow-go/blob/5d2273c199421733efcc5b2a477dfbb6ae085901/engine/common/requester/engine.go#L457) . We just assume that the entity was returned in another request rather than it "had not been requested at all"